src/manager.js

```diff
@@ -701,19 +715,37 @@ AdManager.prototype.refreshSlots = function (slotsToLoad) {
     return;
   }
 
+  var useIAS = typeof this.__iasPET !== 'undefined' && this.options.iasEnabled;
   var useIndex = typeof window.headertag !== 'undefined' && window.headertag.apiReady === true;
   var usePrebid = typeof window.pbjs !== 'undefined' && this.options.prebidEnabled;
+  var refreshMethod = function(){
+    if (useIndex) {
+      return window.headertag.pubads().refresh(slotsToLoad, {changeCorrelator: false});
+    } else if (usePrebid) {
+      return this.prebidRefresh(slotsToLoad);
+    } else {
+      return this.googletag.pubads().refresh(slotsToLoad, {changeCorrelator: false});
+    }
+  }
+
 
-  if (useIndex) {
-    window.headertag.pubads().refresh(slotsToLoad, {
-      changeCorrelator: false
+  if (useIAS){
+    var iasSlotSetup = require('./ias-slot-setup');
```

Comment:
I think this IAS block should be moved into a separate function to prevent `#refreshSlots` from being too bloated